src/lithium/interestingness/timed_run.py

```diff
@@ -156,7 +156,7 @@ def timed_run(commandWithArgs, timeout, logPrefix, inp=None, preexec_fn=None):
     elif return_code == ASAN_EXIT_CODE:
         msg = "CRASHED (Address Sanitizer fault)"
         sta = CRASHED
-    elif return_code > 0:
+    elif 0 < return_code < 0x80000000:
```

Comment:
Update the comment in the `else` block that return_code can now be `>= 0x80000000` in the `else` block.